node/src/command.rs

```diff
@@ -1,43 +1,28 @@
-// Copyright 2019 Joystream Contributors
-// This file is part of Joystream node.
-
-// Joystream node is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-
-// Joystream node is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-
-// You should have received a copy of the GNU General Public License
-// along with Joystream node.  If not, see <http://www.gnu.org/licenses/>.
+// This file is part of Substrate.
+
+// Copyright (C) 2017-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 
 use crate::cli::{Cli, Subcommand};
-use crate::node_executor;
 use crate::{chain_spec, service};
-
-use crate::service::{new_full_base, new_partial, NewFullBase};
-use node_executor::Executor;
-use node_runtime::{opaque::Block, RuntimeApi};
-use sc_cli::{ChainSpec, Result, Role, RuntimeVersion, SubstrateCli};
+use node_runtime::Block;
+use sc_cli::{ChainSpec, Role, RuntimeVersion, SubstrateCli};
 use sc_service::PartialComponents;
 
 impl SubstrateCli for Cli {
     fn impl_name() -> String {
-        "Joystream Node".into()
-    }
-
-    fn support_url() -> String {
-        "https://www.joystream.org/".into()
-    }
-
-    fn copyright_start_year() -> i32 {
-        2019
-    }
-
-    fn executable_name() -> String {
         "joystream-node".into()
```

Comment:
Why did you change the 'impl_name' value?

Reply:
Changes restored https://github.com/Joystream/joystream/pull/2558/commits/9f957769088c3a28cefc76713ad37eddc63d2ee9